src/dace_interface.ts

```diff
@@ -117,10 +117,43 @@ export class DaCeInterface
         });
     }
 
+    /**
+     * Clean out shell commands by ensuring spaces are properly quoted.
+     * Commands containing spaces are surrounded by quotes, and the command
+     * is assembled into command + arguments as one string.
+     * @param cmds      List of command parts, typically command plus arguments.
+     * @param spaceSafe Whether or not to perform space-quoting.
+     * @returns         Assembled command as a string.
+     */
+    private cleanCmd(cmds: string[], spaceSafe = true): string {
+        if (spaceSafe)
+            switch (os.platform()) {
+                case 'win32':
+                    for (let i = 0; i < cmds.length; i++) {
+                        if (/\s/g.test(cmds[i]))
+                            cmds[i] = '& "' + cmds[i] + '"';
```

Comment:
I think this is powershell specific, not Windows specific

Reply:
True, it might need a fallback for CMD, if people use CMD as their default builtin terminal